Source/OIDServiceConfiguration.h

```diff
@@ -76,6 +81,24 @@ typedef void (^OIDServiceConfigurationCreated)
                                 tokenEndpoint:(NSURL *)tokenEndpoint
                          registrationEndpoint:(nullable NSURL *)registrationEndpoint;
 
+/*! @param authorizationEndpoint The authorization endpoint URI.
+    @param tokenEndpoint The token exchange and refresh endpoint URI.
+    @param issuer The OpenID Connect issuer.
+ */
+- (instancetype)initWithAuthorizationEndpoint:(NSURL *)authorizationEndpoint
```

Comment:
Should we mark `init` as unavailable in the header?

Reply:
I think we do already? `- (instancetype)init NS_UNAVAILABLE;`